content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing parts of code to verify its behavior.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-javascript-unit-testing'
+---
+
+Unit testing is an essential part of the coding process. It verifies that all parts of the code work correctly.
+
+A part or unit of code can be anything from a class function to a small snippet of code that performs a specific task. Testing individual components makes it easier to find exactly where an error lies.
+
+There are usually two types of unit testing:
+
+- Manual testing
+- Automated testing
+
+## Manual Testing
+
+Manual testing is when tests are done directly by people. It can involve clicking on objects, providing input, or scrolling through items.
+
+Depending on the length of the software, this type of testing can be time-consuming. An alternative to unit testing would be automated testing.
+
+## Automated Testing
+
+Writing scripts to test parts of code is a way to automate testing. Combined with a testing automation tool, these scripts can run automatically and immediately inform developers if any errors have occurred.
+
+Automated testing is a better way to test code when the process of doing it manually is tedious, frequent testing is needed, or when multiple individuals are adding or making changes to the code.
```

Comment:
@Dusch4593 do you think there is a better way to write your first suggested sentence, "Automated testing is code written to test other code." 

When I first wrote this part, I had something similar to that but changed it .  I thought it sounded a little confusing when read out loud because "code" is in the sentence twice.

Reply:
How about something like this?

"Automated testings are where programs are written specifically to test code."